docs/user-guide/configure.md

```diff
@@ -221,6 +221,14 @@ The value of `"extends"` is a "locater" (or an array of "locaters") that is ulti
 - an absolute path to a file (which makes sense if you're creating a JS object in a Node.js context and passing it in) with a `.js` or `.json` extension.
 - a relative path to a file with a `.js` or `.json` extension, relative to the referencing configuration (e.g. if configA has `extends: "../configB"`, we'll look for `configB` relative to configA).
 
+## `extensions`
```

Comment:
@jeddy3 can you take care of the documentation?
I wouldn't know what needs to be updated.